submissions/rafiatAminu/HTML/html-forms/contact.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
```

Comment:
Kindly delete this file. You already have a contact.html file created in your `html-forms` folder. This one you have here is placed in the wrong folder

Reply:
![Screen Shot 2021-01-12 at 10 08 30 AM](https://user-images.githubusercontent.com/46797474/104311358-5eca1000-54d5-11eb-9e60-0492caafb6b4.png)

Here's a screenshot of my HTML folder. I have no idea why it is displaying that way